homeassistant/components/somfy/cover.py

```diff
@@ -52,15 +60,15 @@ async def async_update(self):
 
     def close_cover(self, **kwargs):
         """Close the cover."""
+        self.cover.close()
```

Comment:
I moved this because we could hit the rate limit and it would have already have set `self._closed`